champions/src/main/resources/configs/skills/skills.yml

```diff
@@ -590,19 +590,8 @@ skills:
     blocktoss:
       enabled: true
       maxlevel: 5
-      cooldown: 10.0
-      cooldownDecreasePerLevel: 1.0
-      baseCharge: 55.0
-      chargeIncreasePerLevel: 15.0
-      baseDamage: 4.0
-      damageIncreasePerLevel: 1.0
-      baseRadius: 4.0
-      radiusIncreasePerLevel: 0.5
-      baseSpeed: 1.4
-      speedIncreasePerLevel: 0.1
-      hitBoxSize: 1.0
-      size: 0.6
-      sizePerLevel: 0.2
+      cooldown: 6.0
```

Comment:
All these should be here.

```
        baseRadius = getConfig("baseRadius", 3.0, Double.class);
        radiusIncreasePerLevel = getConfig("radiusIncreasePerLevel", 0.0, Double.class);
        baseSpeed = getConfig("baseSpeed", 1.5, Double.class);
        speedIncreasePerLevel = getConfig("speedIncreasePerLevel", 0.0, Double.class);
        size = getConfig("size", 0.5, Double.class);
        sizePerLevel = getConfig("sizePerLevel", 0.0, Double.class);
        hitBoxSize = getConfig("hitBoxSize", 1.5, Double.class);
        startSize = getConfig("startSize", 0.1, Double.class);
        displacement = getConfig("displacement", 0.25, Double.class);

Reply:
why

Reply:
consistency, every other skill has a default config. we also want to populate the entire skills.yml upon startup

Reply:
The in-class config IS the defaulty config, this config gets populated no matter what and just leads to unnecessary confusion because you have to separately change two configs

Reply:
kindos already told you to do it anyway just do it